src/interfaces/mastery.ts

```diff
@@ -0,0 +1,12 @@
+export interface MasteryDTO {
```

Comment:
Rename this to `ChampionMasteryDTO` (note the last `DTO`)

![image](https://github.com/javilobo8/riot-api/assets/1013036/c9456d8b-19a0-4bee-b7aa-6d6260c98cc3)